client/src/utils/constants.js

```diff
@@ -27,6 +27,10 @@ export const languages = [
     language: 'Spanish',
     abbreviation: 'es',
   },
+  {
+    language: 'Dutch',
+    abbreviation: 'nl',
+  },
```

Comment:
Can you move this to index `1`? I'd like to keep English at index `0` and the rest in alphabetical order.

Reply:
Done